packages/components/src/components/primitives/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Divider } from './Divider';
+import { DividerOrientations } from './types';
+import { Center, VStack, StackDivider } from '@chakra-ui/react';
+
+export default {
+  component: Divider,
+  title: 'Components/Primitives/Divider',
+  argTypes: {
+    orientation: {
+      options: DividerOrientations,
+      control: { type: 'select' },
+    },
+  },
+} as Meta;
+
+const AllTemplate = () => (
+  <VStack divider={<StackDivider borderColor="gray.200" />} spacing={4}>
```

Comment:
this divider makes hard to identify the actual divider component

Reply:
removed!.